### DISCUSSION TITLE
About making a website for this organisation

### DISCUSSION THREAD

**Discussion by Kittycat58** [TechSpectra/Foodie-Discord-Bot#3] [Ideas]
Lets make a website for this organisation

**Comment by DON2604** [comment]
## Nice suggestion, Yeeah we are thinking about it. Lets work together
![nichijou-anime](https://github.com/TechSpectra/Foodie-Discord-Bot/assets/89240074/100df9c9-b349-4f13-8146-fb358dd69325)
